clusterloader2/run-e2e.sh

```diff
@@ -22,5 +22,18 @@ CLUSTERLOADER_ROOT=$(dirname "${BASH_SOURCE}")
 export KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
 export KUBEMARK_ROOT_KUBECONFIG="${KUBEMARK_ROOT_KUBECONFIG:-${HOME}/.kube/config}"
 
+# Deploy the GCP PD CSI Driver is required
+if [ $1 = "--deploy-csi-driver" ]; then
+   if [[ -z "${E2E_GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
+      echo "Env var E2E_GOOGLE_APPLICATION_CREDENTIALS must be set to deploy driver"
+      exit 1
+   else
+      kubectl create secret generic cloud-sa --from-file="${E2E_GOOGLE_APPLICATION_CREDENTIALS}"
+   fi
+   kubectl apply -f ${CLUSTERLOADER_ROOT}/gcp-csi-driver-stable.yaml
+   kubectl wait pods -l app=gcp-compute-persistent-disk-csi-driver --for condition=Ready --timeout=300s
+   shift
```

Comment:
what does it do?

Reply:
shift shifts the arguments by 1, essentially popping off the first arg. I will remove this for using env vars.